RuddockWebsite/__init__.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, g
+from sqlalchemy import create_engine
+
+from RuddockWebsite import config, constants
+from RuddockWebsite.modules import admin, hassle, users
+
+app = Flask(__name__)
+app.debug = False
```

Comment:
I did this so that, by default, the app has no debugging. This will be how wsgi starts the app. However, `run_server.py` will override this since it is used only for local testing.